services/ui-src/src/components/cards/EntityCard/EntityCard.tsx

```diff
@@ -25,7 +25,21 @@ export const EntityCard = ({
   ...props
 }: Props) => {
   // any drawer-based field will do for this check
-  const entityCompleted = formattedEntityData.population;
+
+  let entityCompleted = false;
+  switch (entityType) {
+    case ModalDrawerEntityTypes.ACCESS_MEASURES:
+      entityCompleted = !!formattedEntityData.population;
+      break;
+    case ModalDrawerEntityTypes.SANCTIONS:
+      // TODO Sanctions
+      break;
+    case ModalDrawerEntityTypes.QUALITY_MEASURES:
+      // TODO quality measures
+      break;
+    default:
+      break;
+  }
```

Comment:
Thought: Would it make sense to pull this out a level and just pass through an entity completion status to the component? Or does it make more sense to keep that logic focused here? Just a thought.

Reply:
we're formatting entity data in the parent file, so would change up the logic a bit. I think this can be reassessed when sanctions and quality measures are done

Reply:
Makes sense to me 👍